### ISSUE TITLE
Accessibility issues

### ISSUE THREAD

**Issue by aidan-fitz** [covid19risk/permission-portal#110]
I found some accessibility problems using the [Microsoft Accessibility Insights tool](https://accessibilityinsights.io/docs/en/web/overview):

- Pages should have [a `<title>` element](https://dequeuniversity.com/rules/axe/3.5/document-title) to aid with navigation.
- Images should have [alt text](https://dequeuniversity.com/rules/axe/3.5/image-alt). Purely decorative images can have an empty one.
- Form elements should have [labels](https://dequeuniversity.com/rules/axe/3.5/label?application=msftAI). For example, on the `/manage_members` page, I would add `aria-label="Role"` or `aria-label="Status"` attributes to each of the dropdown menu items.

**Comment by ibeckermayer** [comment]
Accessibility is not a priority, though you're free to create PR's that add accessibility features if you want.
